pkg/webrtctransport.go

```diff
@@ -77,9 +76,18 @@ func NewWebRTCTransport(ctx context.Context, session *Session, me webrtc.MediaEn
 	session.AddTransport(p)
 
 	pc.OnTrack(func(track *webrtc.Track, receiver *webrtc.RTPReceiver) {
-		log.Debugf("Peer %s got remote track id: %s ssrc: %d rid :%s label: %s", p.id, track.ID(), track.SSRC(), track.RID(), track.Label())
+		log.Debugf("Peer %s got remote track id: %s ssrc: %d rid :%s streamID: %s", p.id, track.ID(), track.SSRC(), track.RID(), track.Label())
 		recv := NewWebRTCReceiver(ctx, track, cfg.router)
 
+		// Set transport label
+		if p.streamID == "" {
```

Comment:
Does this logic assume that a transport can only publish a single "MediaStream"?

Reply:
Yes, it  can be moved to routers, but I could not find a use-case to publish multiple media streams from a single peer.

Reply:
one use case is screen sharing + camera